app/controllers/admin/budget_headings_controller.rb

```diff
@@ -9,6 +9,27 @@ def create
     @headings = @budget_group.headings
   end
 
+  def edit
+    @budget = Budget.find params[:budget_id]
```

Comment:
Our convention is to use parenthesis with find
`Budget.find(params[:budget_id])`